test/system/comment_test.rb

```diff
@@ -321,21 +448,11 @@ def get_path(page_type, path)
     end
 
     test "#{page_type_string}: edit comment" do
+      nodes(node_name).add_comment({
+        uid: 2,
+        body: comment_text
+      })
       visit get_path(page_type, nodes(node_name).path)
-      # Create a comment
-      main_comment_form =  page.find('h4', text: /Post comment|Post Comment/).find(:xpath, '..') # title text on wikis is 'Post comment'
-      # fill out the comment form
```

Comment:
I'm trying to prune the system tests bit-by-bit. Every time we post a comment manually where we don't need to do so is extra runtime. Here `node.add_comment` can save some time.